styles.css

```diff
@@ -0,0 +1,586 @@
+  z-index: 10;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  display: block;
+}
+
+.search__box {
+  margin-top: -2.4rem;
+}
+
+.box {
+  position: relative;
+  width: 6rem;
+  height: 1rem;
+  padding: 6px;
+  text-align: right;
+  margin-top: -2.544rem;
+}
+
+.box::before {
+  content: "";
+  display: block;
+  z-index: 0;
+  top: 0;
+  right: 0;
+  width: 0;
+  height: 0;
+  border: 1.25rem solid transparent;
+  border-bottom-color: white;
+  border-left-color: white;
+  margin-top: -0.3rem;
+  margin-left: -0.5rem;
+}
+
+.content {
+  position: relative;
+  z-index: 1;
+}
+
+.block-title {
+  color: rgba(3, 3, 3, 0.4);
+  font: bolder 20px/16px "Roboto Condensed", sans-serif;
+}
+
+.block-title__debate {
+  color: rgba(3, 3, 3, 0.4);
+  margin-top: 1rem;
+  font-size: 1rem;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: bold;
+}
+
+.block-title__more {
+  color: rgba(3, 3, 3, 0.4);
+  margin-top: 2rem;
+  padding-top: 1rem;
+  margin-bottom: 1rem;
+  font: bolder 20px/16px "Roboto Condensed", sans-serif;
+  border-bottom: 1px solid #ddd;
+}
+
+.block-title__heros {
+  color: rgba(3, 3, 3, 0.4);
+  font-size: 1rem;
+  font-family: "Roboto Condensed", sans-serif;
+  font-weight: bold;
+}
+
+.category {
+  display: block;
+  font: bold 17px/16px "Roboto Condensed", sans-serif;
+  color: #f72210;
+  text-transform: uppercase;
+  padding: 8px 15px 9px;
+  background-color: rgba(255, 255, 255, 0.9);
+  position: absolute;
+  margin-top: -2rem;
+  z-index: 1;
+}
+
+.category__heroes {
+  display: block;
+  position: absolute;
+  margin-top: -1.6rem;
+  padding: 0.3rem;
+  font: bold 0.8rem "Roboto Condensed", sans-serif;
+  color: black;
+  text-transform: uppercase;
+  background-color: #c2d4d7;
+  z-index: 1;
+}
+
+.category__red {
+  display: block;
+  font: bold 17px/16px "Roboto Condensed", sans-serif;
+  color: white;
+  text-transform: uppercase;
+  padding: 8px 15px 9px;
+  background-color: #f72210;
+  position: absolute;
+  margin-top: -2rem;
+  z-index: 1;
+}
+
+.category:hover {
+  color: #f72210;
+  text-decoration: none;
+}
+
+.category__heroes:hover {
+  color: black;
+  text-decoration: none;
+}
+
+.category__red:hover {
+  color: white;
+  text-decoration: none;
+}
+
+.art__tittle {
+  font-size: 20px;
+  line-height: 24px;
+  font-weight: bolder;
+  font-family: "Roboto", sans-serif;
+  margin-bottom: 10px;
+  color: black;
+}
+
+.art__tittle:hover {
+  text-decoration: none;
+  color: black;
+  padding-bottom: 0.1rem;
+  border-bottom: 1px solid red;
+}
+
+.opinion__container img {
+  object-fit: contain;
+}
+
+.debater__one img {
+  object-fit: scale-down;
+  max-height: 90px;
+}
+
+.debater__two img {
+  object-fit: scale-down;
+  max-height: 90px;
+}
+
+.debater__one {
+  border-top: 0.4rem solid red;
+  padding-top: 1rem;
+}
+
+.debater__two {
+  border-bottom: 0.4rem solid #1c87fb;
+  padding-bottom: 1rem;
+}
+
+.debate__art {
+  color: black;
+  font-size: 1rem;
+  font-weight: bold;
+}
```

Comment:
This selector has the same styles as the selector on line 249. Group selectors to minimize code in your CSS file.